### DISCUSSION TITLE
openwrt里的图形化界面，哪个配置优先级更高？

### DISCUSSION THREAD

**Discussion by Nautilus-Erkkinen** [pymumu/smartdns#1511] [Q&A]
比如，我在图形化界面配置端口为6553，在“自定义设置”里面配置端口为7553，哪边配置优先级更高？
或者这么描述：“自定义设置”应该怎么和图形化界面和谐共存？
![image](https://github.com/pymumu/smartdns/assets/43929486/2a44258c-e39e-4a4e-b20d-da32e939f677)

**Comment by PikuZheng** [comment]
以最终生成的配置文件为准，相同配置后加载的生效。
一般情况下，custom.conf在smartdns.conf的尾部，所以“自定义配置”有更高的生效级别。
另一方面，smartdns允许多进程，配置多个监听端口不会冲突，会同时生效。
